JavaScript/packages/recognizers-date-time/src/dateTime/baseMerged.ts

```diff
@@ -39,10 +39,12 @@ export interface IMergedExtractorConfiguration {
 export class BaseMergedExtractor implements IDateTimeExtractor {
     protected readonly config: IMergedExtractorConfiguration;
     protected readonly options: DateTimeOptions;
+    protected readonly dmyDateFormat: boolean;
 
-    constructor(config: IMergedExtractorConfiguration, options: DateTimeOptions) {
+    constructor(config: IMergedExtractorConfiguration, options: DateTimeOptions, dmyDateFormat:boolean = false) {
```

Comment:
Has this argument ever been used anywhere(after the assignment)? I don't think it should be added here.